### ISSUE TITLE
🌟 BART

### ISSUE THREAD

**Issue by Colanim** [huggingface/transformers#1676]
# 🌟New model addition

## Model description

method for pre-training seq2seq models by de-noising text. BART outperforms previous work on a bunch of generation tasks (summarization/dialogue/QA), while getting similar performance to RoBERTa on SQuAD/GLUE

[BART: Denoising Sequence-to-Sequence Pre-training for Natural Language Generation, Translation, and Comprehension](https://arxiv.org/pdf/1910.13461.pdf)

Amazing results on text summarization.

## Open Source status

* [x] the model implementation is available: not yet
* [x] the model weights are available: not yet
* [ ] who are the authors: @yinhanliu @ernamangoyal

**Comment by tahmedge** [comment]
They mentioned that they also applied their model in the recently published ELI5 dataset. Do you know how did they give the question along with the passage as input to find out the abstractive answer? Because for the CNN-DailyMail dataset, it did not consider any questions.

**Comment by rlouf** [comment]
@thomwolf another encoder-decoder

**Comment by stefan-it** [comment]
Was released today: https://github.com/pytorch/fairseq/tree/master/examples/bart 🎉

**Comment by Colanim** [comment]
@stefan-it is it the official release ?

**Comment by yinhanliu** [comment]
Let me know if you guys plan to add xsum/eli5/cnn-dm ft with our released bart into hugging face.

**Comment by nicola-decao** [comment]
Is there any news on this?

**Comment by hamediramin** [comment]
any progress on this one? also thanks :)

**Comment by sshleifer** [comment]
I'm getting started on this Feb 4!
